.github/workflows/publish.yml

```diff
@@ -24,10 +24,6 @@ jobs:
             target: arm-unknown-linux-gnueabihf
             use-cross: true
 
-          - os: windows-latest
-            target: x86_64-pc-windows-msvc
-            use-cross: false
```

Comment:
I'd prefer if this is swapped for just commenting out the lines, so that I remember to get it working again (can't add a suggestion on deleted lines unfortunately)